Meridian59/Data/Models/AdminData/AdminData.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Meridian59.Common.Interfaces;
+using Meridian59.Data.Lists;
+using Meridian59.Protocol.Events;
+using Meridian59.Protocol.GameMessages;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    public class AdminData : INotifyPropertyChanged, IClearable
+    {
+        //Handles the objects we're watching in the AdminUI
+        private BaseList<AdminObject> TrackedObjects { get; set; }
+        public void TrackAdminObject(AdminObject adminObject)
+        {
+            TrackedObjects.Add(adminObject);
+            OnTrackedAdminObjectAdded(new AddTrackedAdminObjectEventHandlerArgs(this, adminObject));
```

Comment:
You should not need an event like "AdminObjectAdded".
The BaseList class of "TrackedObjects" already provides an event named "ListChanged".

Just subscribe this with an handler and check for type "ItemAdded".
This handler will be triggered by the line:
>> TrackedObjects.Add(adminObject);